repository-3.0.json

```
@@ -1,5 +1,47 @@
 {
     "drivers": [
+        {
+        	"name": "chromedriver",
+        	"platform": "windows",
+        	"bit": "32",
+        	"version": "126.0.6478.126",
+        	"url": "https://storage.googleapis.com/chrome-for-testing-public/126.0.6478.126/win32/chrome-win32.zip",
```

We need the `chromedriver` binary as opposed to `chrome`
```suggestion
        	"url": "https://storage.googleapis.com/chrome-for-testing-public/126.0.6478.126/win32/chromedriver-win32.zip",
```